openassessment/xblock/job_sample_grader/code_grader.py

```diff
@@ -93,6 +103,27 @@ def get_error_response(run_type, error):
         }
 
     def run_code(self, run_type, lang, code_file_name, lang_extension_file_path, problem_name):
+        """
+        Extract test cases for given problem. Sort test cases based on test number.
+        Execute code and compare output with test case out. Update response based on
+        correctness of test case.
```

Comment:
This feels like a description of the code itself. Docstrings should explain what the function/method is doing without delving into the code parts.